modules/ROOT/pages/dashboard-config-ref.adoc

```diff
@@ -30,7 +30,7 @@ Many <<metrics_basic, basic>> and <<metrics_advanced, advanced>> metrics are ava
 [[metrics_basic]]
 === Basic Mode Metrics for Applications
 
-These metrics are supported for the following Mule runtime engine versions: 3.8.7-AM, 3.9.x, 4.1.x, and 4.2.0.
+These metrics are supported for the following Mule runtime engine versions: 3.8.7, 3.9.x, 4.1.x, and 4.2.x.
```

Comment:
"Mule runtime engine versions": Couldn't that noun string be simply "versions of Mule Runtime"?

Reply:
```suggestion
These metrics are supported for the following versions of Mule Runtime: 3.8.7, 3.9.x, 4.1.x, and 4.2.x.
```